workday-application/src/main/kotlin/community/flock/eco/workday/application/controllers/ExpenseController.kt

```diff
@@ -126,69 +145,68 @@ class ExpenseController(
         documentService
             .storeDocument(file.bytes)
             .toResponse()
-}
```

Comment:
These seem to be the old implementation still. Should we convert this to wirespec to just to be sure?